templates.go

```diff
@@ -178,5 +178,41 @@ func getTemplates() *template.Template {
 	if err != nil {
 		logger.Fatalf("failed parsing template %s", err)
 	}
+
+//<!--COCO Begin-->
+//COCO: Add logout html page to call logout API
+//<!--COCO End-->
+        t, err = t.Parse(`{{define "logout.html"}}
+<!DOCTYPE html>
+<html lang="en" charset="utf-8">
+<head>
+        <title>{{.Title}}</title>
+</head>
+<script>
+function logout() {
```

Comment:
- ensure that the alignment is matching the other code in the file.

Reply:
handled